ibind/client/ibkr_client_mixins/session_mixin.py

```diff
@@ -32,7 +32,7 @@ def initialize_brokerage_session(self: 'IbkrClient', publish: bool, compete: boo
             publish (Boolean): Determines if the request should be sent immediately. Users should always pass true. Otherwise, a ‘500’ response will be returned.
             compete (Boolean): Determines if other brokerage sessions should be disconnected to prioritize this connection.
         """
-        return self.post('iserver/auth/ssodh/init', {'publish': publish, 'compete': compete})
+        return self.post(path='iserver/auth/ssodh/init', params={'publish': publish, 'compete': compete})
```

Comment:
path and params are the first two arguments in `post`. Any reason why we'd want to make them keyword arguments like this?

Reply:
No real need, I was being pedantic to try and find the bugs.